### PR TITLE
Added WSL support

### PR DESCRIPTION
**Pull request by nxht**
Hi, your plugin didn't work as expected on WSL environment and I've tried to make it work.

One thing to consider is that it uses $WSL_DISTRO_NAME environment variable to detect the WSL distro name which is available since WSL builds 18362 (WSL2). So it might not work for older builds.

### CODE REVIEW

**Comment by sandcastle** [comment]
@Indlgo - thanks for the work on this mate. My wife is terminally ill atm, so don't have much time to look at it - have you done much testing with your version? Happy to merge in if you think its stable

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-# vscode-open
+# Open with Default Application (WSL supported)
 
-> Opens files using the OS's default program for the file type
+> Opens files using the OS's (in Windows for WSL) default program for the file type
+
+ - This is forked version of [open by sandcastle](https://marketplace.visualstudio.com/items?itemName=sandcastle.vscode-open) with WSL support added
```
Comment (sandcastle) [inline README.md:5]: Remove this line of the comment - this is not a forked version

File: package.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-  "name": "vscode-open",
+  "name": "vscode-open-wsl",
   "displayName": "Open",
-  "description": "Opens the current file with the default for the OS",
-  "version": "0.2.0",
+  "description": "Opens the current file with the default application for the OS (in Windows for WSL)",
+  "version": "0.2.1",
   "icon": "open.png",
-  "publisher": "sandcastle",
+  "publisher": "IndlgO",
```
Comment (sandcastle) [inline package.json]: Change back to @sandcastle

File: package.json [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
   "engines": {
     "vscode": "^1.50.0"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/sandcastle/vscode-open"
+    "url": "https://github.com/Indlgo/vscode-open"
```
Comment (sandcastle) [inline package.json:13]: Change back to `sandcastle`

File: package.json [hunk 4]
```diff
@@ -19,42 +19,42 @@
     "display",
     "vscode"
   ],
-  "homepage": "https://github.com/sandcastle/vscode-open",
+  "homepage": "https://github.com/Indlgo/vscode-open",
```
Comment (sandcastle) [inline package.json:22]: Change back to `sandcastle`

File: package.json [hunk 5]
```diff
@@ -19,42 +19,42 @@
...
   "bugs": {
-    "url": "https://github.com/sandcastle/vscode-open/issues"
+    "url": "https://github.com/Indlgo/vscode-open/issues"
```
Comment (sandcastle) [inline package.json:24]: Change back to `sandcastle`

File: README.md [hunk 6]
```diff
@@ -1,6 +1,8 @@
-# vscode-open
+# Open with Default Application (WSL supported)
 
-> Opens files using the OS's default program for the file type
+> Opens files using the OS's (in Windows for WSL) default program for the file type
```
Comment (sandcastle) [inline README.md:3]: It probably reads better as 

> Opens files using the OS's default program for the file type, including support for WSL on Windows`

**Review by sandcastle — COMMENTED** [review]
Just minor doco tweaks required

**Comment by nxht** [comment]
Beside the first commit (https://github.com/sandcastle/vscode-open/pull/17/commits/4395c0ddf629f7c6caa2ec72c94e71316f3ec7be), please ignore all other changes.

I needed to use the WSL-related feature and as I thought you're inactive, I published forked version separately. All commits after the first are related to it.

**Pull request closed without merging**